shell/common/shell_unittests.cc

```diff
@@ -1014,5 +1015,114 @@ TEST_F(ShellTest, Screenshot) {
   DestroyShell(std::move(shell));
 }
 
+static bool MemsetPatternSetOrCheck(uint8_t* buffer,
+                                    size_t size,
+                                    bool set_or_check) {
+  if (buffer == nullptr) {
+    return false;
+  }
+
+  auto pattern4 = reinterpret_cast<const uint8_t*>("dErP");
+  uint8_t* start = buffer;
+  uint8_t* p = buffer;
+
+  while ((start + size) - p >= 4) {
```

Comment:
nit: Pretty obvious what this is, but maybe `pattern` and a `constexpr size_t pattern_len = 4;`?

Reply:
Done.